django/cantusdb_project/main_app/management/commands/merge_indexer_user.py

```diff
@@ -20,7 +20,10 @@ def handle(self, *args, **options):
             # do not use the existing "name" property, 
             # because for some reason, the first/last names can have trailing spaces
             # TODO: populate the full_name field and remove the "name" property
```

Comment:
remove this comment? (perhaps you do it in a future commit)

Reply:
This is still a TODO. I need to change everything else that depends on this `name` property. I think using the `full_name` field instead is better.